README.md

-   [BTS](#bts)
-   [Installation and use](#installation-and-use)
-   [Data sets](#data-sets)
-   [Errata](#errata)

<!-- README.md is generated from README.Rmd. Please edit that file -->
<!-- ANLEITUNG FÜR MARKUS UND PETER -->
<!-- Zwei Schritte: -->
<!-- 1. In der Datei data-raw/DATASET.R Kommandos für das Holen der Daten ausführen. Am Ende über den Befehl usethis::use_data(...) Daten im Paket zur Verfügung stellen. Ich hab meine Dateien auch als csv gespeichert, sodass man sie gebündelt hat. -->
<!-- 2. In der Datei R/data.R Datensatz dokumentieren -->
<!-- Daten sind dann am Ende im bts Paket (momentan auf github), Installation beispielsweise über devtools::install_github("johannes-titz/bts") -->
<!-- Aufruf der Daten über data(...) nachdem bts geladen ist. Siehe auch die folgende Anleitung.-->

## BTS

The package provides data sets for the book “Datenanalyse mit R:
Fortgeschrittene Verfahren”, Burkhardt, Titz, Sedlmeier, Pearson Studium
Psychologie, (ISBN-10: 3868944133, ISBN-13: 978-3868944136)

## Installation and use

Install it via devtools (choose your own repo):

``` r
install.packages("devtools", repos = "https://ftp.fau.de/cran/")
devtools::install_github("johannes-titz/bts")
```

or via librarian (choose your own repo):

``` r
install.packages("librarian", repos = "https://ftp.fau.de/cran/")
librarian::shelf(johannes-titz/bts)
```

To load the data, for instance icecream:

``` r
library(bts)
data(icecream)
head(icecream)
#>   Observations    Flavor       Packaging      Light     Organic   individual
#> 1    Profile 1 Raspberry Homemade waffle No low fat Not organic Individual 1
#> 2    Profile 1 Raspberry Homemade waffle No low fat Not organic Individual 2
#> 3    Profile 1 Raspberry Homemade waffle No low fat Not organic Individual 3
#> 4    Profile 1 Raspberry Homemade waffle No low fat Not organic Individual 4
#> 5    Profile 1 Raspberry Homemade waffle No low fat Not organic Individual 5
#> 6    Profile 1 Raspberry Homemade waffle No low fat Not organic Individual 6
#>   ranking
#> 1       1
#> 2       6
#> 3       5
#> 4       1
#> 5       2
#> 6       7
```

## Data sets

Currently available data sets are:

``` r
data(package = "bts")
```

| Item            | Title                                                           |
|:----------------|:----------------------------------------------------------------|
| Arbeit          | Befragung zur Arbeitszufriedenheit                              |
| FW_abbrecher    | Fehlende Werte 3 (Missing Values 3)                             |
| FW_mar          | Fehlende Werte 2 (Missing Values 2)                             |
| FW_mcar         | Fehlende Werte 1 (Missing Values 1)                             |
| Lehr_Lern       | Mediation von Lernleistung (Meadiation of learning achievement) |
| Meditation      | 6 Meditationsstudien (6 Studys of Meditation)                   |
| Mehrere_AVs     | Mehrere AVs in einem Einzelfall                                 |
| Obst            | Präferenz für Früchte (fruit rating)                            |
| Power_Pose      | Power_Pose                                                      |
| St_dist_aggr    | Straftaten agreggierte Distanzen                                |
| St_dist_ind     | Individuelle Distanzen von Straftaten                           |
| Stadt_dist      | Distanzen von 5 deutschen Städten                               |
| Straftaten      | Straftaten                                                      |
| VL_17           | VL_17                                                           |
| Wellbeing_Daten | 6 Meditierende im Multiple Baseline Design (scan-Paket)         |
| Wellbeing_kurz  | Lebenszufriedenheit von 6 Einzelfällen                          |
| hdp             | Hospital Doctor Patient Dataset                                 |
| icecream        | Ice cream rankings                                              |
| pedersen_2002   | pedersen_2002                                                   |
| pub_bias0       | dataset without publication bias                                |
| pub_bias1       | A dataset with publication bias                                 |
| school          | School data set                                                 |

Data sets in bts

To get information about a data set use

``` r
help(icecream)
Ice cream rankings

Description:

     A dataset containing ice cream attributes and rankings from 15
     (fictitious) participants.

Usage:

     icecream
     
Format:

     A data frame with 150 rows and 7 variables:

     Observations profile number

     Flavor flavor of ice cream

     Packaging packaging of ice cream

     Light fat content of ice cream

     Organic is ice cream organic?

     individual ID for participant

     ranking ranking of ice cream, from 1 to 10

Source:

     <URL:
     https://help.xlstat.com/s/article/conjoint-analysis-in-excel-tutorial-new?language=en_US>
     <URL:
     http://users.telenet.be/samuelfranssens/tutorial_data/icecream.xlsx>
```

## Errata


NEWS.md
# bts

# bts 0.1.0

* first release


data-raw/DATASET.R
## code to prepare datasets 

################################################
# Chapter MEA
# ice cream
download.file("http://users.telenet.be/samuelfranssens/tutorial_data/icecream.xlsx",
              "iscreamforicecream.xlsx")
icecream <- readxl::read_xlsx("iscreamforicecream.xlsx")
library(tidyr)
icecream <- as.data.frame(
  pivot_longer(icecream, cols = 6:20, names_to = "individual",
               values_to = "ranking")
)
write.csv(icecream, "data-raw/icecream.csv")
usethis::use_data(icecream, overwrite = TRUE)

# hdp
hdp <- read.csv("https://stats.idre.ucla.edu/stat/data/hdp.csv")
hdp <- hdp[, c("remission", "CancerStage", "Experience", "DID")]
write.csv(hdp, "data-raw/hdp.csv")
usethis::use_data(hdp, overwrite = TRUE)

# school
school <- read.table(
  "http://bayes.acs.unt.edu:8083/BayesContent/class/Jon/R_SC/Module9/lmm.data.txt",
  header = TRUE, sep = ",", na.strings = "NA", dec = ".",
  strip.white = TRUE
)
write.csv(school, "data-raw/school.csv")
usethis::use_data(school, overwrite = TRUE)

######################################
## Chapter: Missing Values

# FW mcar
FW_mcar <- read.csv2("https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/FW_mcar.csv")
write.csv(FW_mcar, "data-raw/FW_mcar.csv")
usethis::use_data(FW_mcar, overwrite = TRUE)

# FW mar
FW_mar <- read.csv2("https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/FW_mar.csv")
write.csv(FW_mar, "data-raw/FW_mar.csv")
usethis::use_data(FW_mar, overwrite = TRUE)

# FW abbrecher
FW_abbrecher <- read.csv2("https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/FW_abbrecher.csv")
write.csv(FW_abbrecher, "data-raw/FW_abbrecher.csv")
usethis::use_data(FW_abbrecher, overwrite = TRUE)

######################################
## Chapter 08 MDS

# Stadt_dist
library(readr)
Stadt_dist <- read_delim(
  "https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/Stadt_dist.csv",
  delim = ";", escape_double = FALSE, locale = locale(encoding = "ISO-8859-1"),
  trim_ws = TRUE
)

Stadt_dist <- data.frame(Stadt_dist[, 2:6])
row.names(Stadt_dist) <- colnames(Stadt_dist)
write.csv(Stadt_dist, "data-raw/Stadt_dist.csv")
usethis::use_data(Stadt_dist, overwrite = TRUE)

# Obst
Obst <- data.frame(Apfel = c(8, 7, 5, 2),
                   Birne = c(6, 7, 4, 4),
                   Kirsche = c(4, 6, 6, 6),
                   Aprikose = c(3, 2, 5, 7))
write.csv(Obst, "data-raw/Obst.csv")
usethis::use_data(Obst, overwrite = TRUE)

# Straftaten
Straftaten <- read.csv2("https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/Straftaten.csv")
write.csv(Straftaten, "data-raw/Straftaten.csv")
usethis::use_data(Straftaten, overwrite = TRUE)

# St_dist_aggr
St_dist_aggr <- read.csv(
  "https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/St_dist_aggr.csv",
  row.names = 1
)
write.csv(St_dist_aggr, "data-raw/St_dist_aggr.csv")
usethis::use_data(St_dist_aggr, overwrite = TRUE)

# St_dist_ind
get_distances_from_row <- function(sep_row, allnames) {
  dist_list <- NULL
  for (i in seq_along(allnames)) {
    pat <- allnames[i]
    # Extrahiere die Werte anhand der jeweiligen Straftat und füge die 0
    # (die Ähnlichkeit der Straftat mit sich selbst) hinzu.
    pat_means <- c(sep_row[grep(pat, names(sep_row))], 0)
    # Extrahiere die Namen der Straftaten
    allnames2 <- c(
      unlist(lapply(
        strsplit(names(sep_row[grep(pat, names(sep_row))]), split = pat),
        function(x) x[length(x)])
      ), pat)
    # Füge die korrekten Bezeichnungen hinzu
    names(pat_means) <- allnames2
    # ordne die Werte alphabetisch nach der Bezeichnung
    dist_list[[i]] <- pat_means[order(names(pat_means))]
  }
  # Speichere die Liste als Distanzmatrix
  dists <- as.dist(do.call(cbind, dist_list), upper = TRUE)
  return(dists)
}
# Erstellung der Dsitanzmatrix
allnames <- c("Ein", "Koe", "Rau", "Sac",  "Ste", "Tru", "Unt", "Ver", "Wah",
              "Wid")
St_dist_ind <- NULL
for (i in 1:30) {
  St_dist_ind[[i]] <- get_distances_from_row(
    Straftaten[i, ],
    allnames = allnames)
}
usethis::use_data(St_dist_ind, overwrite = TRUE)

######################################
## Chapter: Pfadanalysen
####################################
# ## generate non-scaled data
library(MASS)
library(Matrix)
ms <- c(0, 0, 0) ### specify means
S1 <- matrix(c(10, 8.6, 7.4,
               8.6, 10, 7.5,
               7.4, 7.5, 10
) / 10,
ncol = 3, byrow = TRUE)
# create data
set.seed(123122)
d <- as.data.frame(mvrnorm(n = 90, mu = ms, S1, tol = .5))
round(cor(d), 2)
Motivation <- c(98, 98, 103, 101, 100, 106, 111, 125, 120, 115)
Motivation <- c(c(98, 98, 103, 101, 100, 106, 111, 125, 120, 115),
                round(d$V1 * sd(Motivation) + mean(Motivation)))

Lernleistung <- c(4, 5, 6, 7, 8, 10, 11, 12, 14, 15)
Lernleistung <- abs(c(4, 5, 6, 7, 8, 10, 11, 12, 14, 15,
                      round(d$V2 * sd(Lernleistung) + mean(Lernleistung))))

Unterrichtsguete <- c(5, 6.5, 7, 5.5, 9, 6.5, 8, 10.5, 8, 10)
Unterrichtsguete <- c(c(5, 6.5, 7, 5.5, 9, 6.5, 8, 10.5, 8, 10),
                      round(d$V3 * sd(Unterrichtsguete) + mean(Unterrichtsguete),
                            1))

Lehr_Lern <- data.frame(Motivation, Lernleistung, Unterrichtsguete)
Lehr_Lern <- Lehr_Lern[1:30, ]
write.csv(Lehr_Lern, "data-raw/Lehr_Lern.csv")
usethis::use_data(Lehr_Lern, overwrite = TRUE)

Power_Pose <- read.csv2("https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/Power_Pose.csv")
write.csv(Power_Pose, "data-raw/Power_Pose.csv")
usethis::use_data(Power_Pose, overwrite = TRUE)

######################################
## Chapter 02: Metaanalyse

# Meditation
Meditation <- data.frame(
  Studie = c("Geary_2011", "Carissoli_2015", "Kang_2014", "Shearer_2016a",
             "Shearer_2016b", "Greeson_2014"),
  Effekt_r = c(.32, -.05, .18, .5, .39, .22),
  N = c(108, 38, 68, 53, 47, 90),
  AV = c(rep(c("Fragebogen", "Heart Rate Variablity"), c(3,2)), "Fragebogen")
)
write.csv(Meditation, "data-raw/Meditation.csv")
usethis::use_data(Meditation, overwrite = TRUE)

# craate Data for publicationbias
set.seed(546451)
Pop1 <- rnorm(20000, 0, 1)
Pop2 <- rnorm(20000, .5, 1)
k  <- 56
N <- round(rnorm(k - 10, 40, 10))
N <- c(N,round(runif(10, 100, 200) ))
M1 <- matrix(NA, ncol = 3, nrow = k)
for(i in 1:k){
  ix <- sample(1:40000, N[i])
  r <- cor(c(Pop1, Pop2)[ix],
           ifelse(ix <= 20000, 0,1)
  )
  p <- cor.test(c(Pop1, Pop2)[ix],
                ifelse(ix <= 20000, 0,1))$p.value

  M1[i, 1] <- N[i]
  M1[i, 2] <- r
  M1[i, 3] <- round(p, 3)
}
pub_bias0 <- data.frame(M1)
pub_bias1 <- data.frame(M1[which(M1[, 3] <= 0.05),])
colnames(pub_bias0) <- c("N", "r", "p")
colnames(pub_bias1) <- c("N", "r", "p")
write.csv(pub_bias0, "data-raw/pub_bias0.csv")
usethis::use_data(pub_bias0, overwrite = TRUE)
write.csv(pub_bias1, "data-raw/pub_bias1.csv")
usethis::use_data(pub_bias1, overwrite = TRUE)

######################################
## Chapter: Strukturgleichungsmodelle
library(readr)
Arbeit <- read_delim("data-raw/Arbeit.csv",
                     delim = ";", escape_double = FALSE, trim_ws = TRUE)
write.csv(Arbeit, "data-raw/Arbeit.csv")
usethis::use_data(Depression, overwrite = TRUE)

##### MB
St_dist_aggr <- read.csv(
  "https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/St_dist_aggr.csv",
  row.names = 1
)
Straftaten <- read.csv2("https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/Straftaten.csv")
####################################################################


####################################################################
# Chapter 03: Resampling-Verfahren

# Pedersen
pedersen_2002 <- read.csv("https://www.pearson-studium.de/einfuehrung_r/pedersen_2002.txt", sep = "")
write.csv(pedersen_2002, "data-raw/pedersen_2002.csv")
usethis::use_data(pedersen_2002, overwrite = TRUE)

VL_17 <- read.csv("https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/VL17.csv")
write.csv(VL_17, "data-raw/VL_17.csv")
usethis::use_data(VL_17, overwrite = TRUE)

####################################################################
# Chapter 9: Einzelfallanalyse
Mehrere_AVs <- read.csv("https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/Mehrere_AVs.csv")
write.csv(Mehrere_AVs, "data-raw/Mehrere_AVs.csv")
usethis::use_data(Mehrere_AVs, overwrite = TRUE)

library(scan)
Wellbeing_Daten <- readSC(
  "https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/wellbeing_kurz.csv",
  cvar = "case", pvar = "phase",
  dvar = "wellbeing", mvar = "study_day",
  phase.names = c("A", "B")
)
usethis::use_data(Wellbeing_Daten, overwrite = TRUE)

Wellbeing_kurz <- read.csv("https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/wellbeing_kurz.csv")
write.csv(Wellbeing_kurz, "data-raw/wellbeing_kurz.csv")
usethis::use_data(Wellbeing_kurz, overwrite = TRUE)

###############################################################################
# Chapter 5 Strukturgleichungsmodelle

Arbeit <- read.csv2("https://www-user.tu-chemnitz.de/~burma/TUC_R/Band2/Arbeit.csv")
write.csv(Arbeit, "data-raw/Arbeit.csv")
usethis::use_data(Arbeit, overwrite = TRUE)


R/data.R
#' Ice cream rankings
#'
#' A dataset containing ice cream attributes and rankings from 15 (fictitious)
#' participants.
#'
#' @format A data frame with 150 rows and 7 variables:
#' \describe{
#'   \item{Observations}{profile number}
#'   \item{Flavor}{flavor of ice cream}
#'   \item{Packaging}{packaging of ice cream}
#'   \item{Light}{fat content of ice cream}
#'   \item{Organic}{is ice cream organic?}
#'   \item{individual}{ID for participant}
#'   \item{ranking}{ranking of ice cream, from 1 to 10}
#'   }
#' @source
#'   \url{https://help.xlstat.com/s/article/conjoint-analysis-in-excel-tutorial-new?language=en_US}
#'   \url{http://users.telenet.be/samuelfranssens/tutorial_data/icecream.xlsx}
"icecream"

#' Hospital Doctor Patient Dataset
#'
#' A dataset containing simulated data on lung cancer in a
#' three-level-hierarchical structure
#'
#' @format A data frame with 8525 rows and 4 variables:
#' \describe{
#'   \item{remission}{cancer in remission? (TRUE/FALSE)}
#'   \item{CancerStage}{stage of cancer}
#'   \item{Experience}{experience of doctor}
#'   \item{DID}{doctor ID}
#' }
#' @source \url{https://stats.idre.ucla.edu/r/codefragments/mesimulation/}
"hdp"

#' School data set
#'
#' A dataset containing simulated data on students in different schools. Note
#' that the variable names are not important as this is simulated data.
#'
#' @format A data frame with 1200 rows and 7 variables:
#' \describe{
#'   \item{id}{student ID}
#'   \item{extro}{probably extroversion}
#'   \item{open}{probably openness for experience}
#'   \item{agree}{probably agreeableness}
#'   \item{social}{?}
#'   \item{class}{ID for class}
#'   \item{school}{ID for school}
#'   }
#' @source
#' \url{http://bayes.acs.unt.edu:8083/BayesContent/class/Jon/R_SC/Module9/LMM_Examples.R}
#' \url{http://bayes.acs.unt.edu:8083/BayesContent/class/Jon/R_SC/Module9/lmm.data.txt}
"school"

#' Fehlende Werte 1 (Missing Values 1)
#'
#' A fictitious dataset containing 4 variables with 30 values missing completly at random.
#'
#' @format A data frame with 100 rows and 4 variables:
#' \describe{
#'   \item{IQ}{IQ}
#'   \item{Alter}{Alter}
#'   \item{LZ}{Lebenszufriedenheit}
#'   \item{KFZ}{Besitzen Sie ein Auto?}
#'   }
"FW_mcar"

#' Fehlende Werte 2 (Missing Values 2)
#'
#' A fictitious dataset containing 4 variables with 45 values missing at random.
#'
#' @format A data frame with 100 rows and 4 variables:
#' \describe{
#'   \item{IQ}{IQ}
#'   \item{Alter}{Age}
#'   \item{LZ}{life satisfaction}
#'   \item{KFZ}{car}
#'   }
"FW_mar"

#' Fehlende Werte 3 (Missing Values 3)
#'
#' A fictitious dataset containing 4 variables and a typical pattern of missing values which indicates a break off.
#'
#' @format A data frame with 100 rows and 4 variables:
#' \describe{
#'   \item{IQ}{IQ}
#'   \item{Alter}{Age}
#'   \item{LZ}{life satisfaction}
#'   \item{KFZ}{car}
#'   }
"FW_abbrecher"

#' Distanzen von 5 deutschen Städten
#'
#' Stadt_dist ist eine Matrix mit den Entfernungen (Luftlinie in km) zwischen
#' 5 deutschen Städten.
#'
#' @format A symmetric data.frame with 5 rows.
"Stadt_dist"

#' Präferenz für Früchte (fruit rating)
#'
#' A fictitious data set with preferences for four different fruits.
#'
#' @format A data frame with 4 rows and 4 variables:
#' \describe{
#'   \item{Apfel}{Rating of apple}
#'   \item{Birne}{Rating of pear}
#'   \item{Kirsche}{Rating of cherry}
#'   \item{Aprikose}{Rating of apricot}
#'   }
"Obst"

#' Straftaten
#'
#' A data set with similarity ratings for ten criminal offenses measured by pairwise comparison.
#'
#' @format A data frame with 30 rows and 45 variables:
#' \describe{
#'   \item{UntSac}{similaryt between failure to assist a person in danger and damage to property}
#'   \item{SteTru}{similaryt between tax evasion and drunk driving}
#'   \item{SacWid}{similaryt between damage to property and resistance against enforcement officers}
#'   \item{RauSte}{similaryt between misappropriation and tax evasion}
#'   \item{SteUnt}{similaryt between tax evasion and failure to assist a person in danger}
#'   \item{WidKoe}{similaryt between resistance against enforcement officers and damage to property}
#'
#'   \item{SteWid}{similaryt between tax evasion and resistance against enforcement officers}
#'   \item{EinTru}{similaryt between burglary and drunk driving}
#'   \item{KoeSac}{similaryt between misappropriation and damage to property}
#'   \item{SteEin}{similaryt between tax evasion and burglary}
#'   \item{EinKoe}{similaryt between burglary and damage to property}
#'   \item{VerWid}{similaryt between rape and resistance against enforcement officers}
#'   \item{WidRau}{similaryt between resistance against enforcement officers and damage to property}
#'
#'   \item{UntWid}{similaryt between failure to assist a person in danger and resistance against enforcement officers}
#'   \item{KoeTru}{similaryt between misappropriation and drunk driving}
#'   \item{EinUnt}{similaryt between burglary and failure to assist a person in danger}
#'   \item{TruSac}{similaryt between drunk driving and damage to property}
#'   \item{UntVer}{similaryt between failure to assist a person in danger and rape}
#'   \item{TruUnt}{similaryt between drunk driving and failure to assist a person in danger}
#'
#'   \item{WahSac}{similaryt between misappropriation and damage to property}
#'   \item{WidTru}{similaryt between resistance against enforcement officers and drunk driving}
#'   \item{TruVer}{similaryt between drunk driving and rape}
#'   \item{KoeSte}{similaryt between misappropriation and tax evasion}
#'   \item{RauUnt}{similaryt between misappropriation and failure to assist a person in danger}
#'   \item{KoeRau}{similaryt between misappropriation and damage to property}
#'   \item{WidEin}{similaryt between resistance against enforcement officers and burglary}
#'
#'   \item{SteWah}{similaryt between tax evasion and damage to property}
#'   \item{KoeVer}{similaryt between misappropriation and damage to property}
#'   \item{VerWah}{similaryt between misappropriation and damage to property}
#'   \item{EinWah}{similaryt between burglary and damage to property}
#'   \item{RauEin}{similaryt between misappropriation and burglary}
#'   \item{UntWah}{similaryt between failure to assist a person in danger and damage to property}
#'
#'   \item{SacVer}{similaryt between damage to property and rape}
#'   \item{RauWah}{similaryt between misappropriation and damage to property}
#'   \item{WahKoe}{similaryt between misappropriation and damage to property}
#'   \item{RauTru}{similaryt between misappropriation and drunk driving}
#'   \item{SteVer}{similaryt between tax evasion and rape}
#'   \item{SacEin}{similaryt between damage to property and burglary}
#'   \item{RauVer}{similaryt between misappropriation and rape}
#'
#'   \item{WahWid}{similaryt between misappropriation and resistance against enforcement officers}
#'   \item{RauSac}{similaryt between misappropriation and damage to property}
#'   \item{TruWah}{similaryt between drunk driving and damage to property}
#'   \item{KoeUnt}{similaryt between misappropriation and failure to assist a person in danger}
#'   \item{VerEin}{similaryt between rape and burglary}
#'   \item{SteSac}{similaryt between tax evasion and damage to property}
#'   }
"Straftaten"

#' Straftaten agreggierte Distanzen
#'
#' A distance matrix with mean aggregated distances for ten criminal offenses
#'
#' @format A distance matrix with 10 objects:
#' \describe{
#'   \item{Unt}{failure to assist a person in danger}
#'   \item{Ste}{tax evasion}
#'   \item{Sac}{damage to property}
#'   \item{Rau}{misappropriation}
#'   \item{Wid}{resistance against enforcement officers}
#'   \item{Tru}{drunk driving}
#'   \item{Sac}{damage to property}
#'   \item{Ein}{burglary}
#'   \item{Ver}{rape}
#'   \item{Koe}{bodily injury}
#'   }
"St_dist_aggr"


#' Individuelle Distanzen von Straftaten
#'
#' A list of 30 distance matrices with distances for ten criminal offenses.
#' Each matrix represents a individual rating
#'
#' @format A distance matrix with 10 objects:
#' \describe{
#'   \item{Unt}{failure to assist a person in danger}
#'   \item{Ste}{tax evasion}
#'   \item{Sac}{damage to property}
#'   \item{Rau}{misappropriation}
#'   \item{Wid}{resistance against enforcement officers}
#'   \item{Tru}{drunk driving}
#'   \item{Sac}{damage to property}
#'   \item{Ein}{burglary}
#'   \item{Ver}{rape}
#'   \item{Koe}{bodily injury}
#'   }
"St_dist_ind"

#' Mediation von Lernleistung (Meadiation of learning achievement)
#'
#' A fictitious dataset containing 3 variables for mediation
#'
#' @format A data frame with 30 rows and 3 variables:
#' \describe{
#'   \item{Motivation}{Motivation}
#'   \item{Lernleistung}{Lernleistung}
#'   \item{Unterrichtsguete}{Unterrichtsgüte}
#'   }
"Lehr_Lern"

#' Power_Pose
#'
#' A dataset of 16 cases beeing part of an powerposing experiment.
#'
#' @format A data frame with 10 rows and 3 variables:
#' \describe{
#'   \item{Pose}{Experimental condition: high powerpose vs. low powerpose}
#'   \item{Dominanz}{Dominance rating by the neutral obeserver}
#'   \item{Aktivitaet}{Activity}
#'   \item{Fuehrung}{External rating of leadership}
#'   \item{Moderator}{Moderation effect of Dom x Pose}
#'   }
"Power_Pose"

#' 6 Meditationsstudien (6 Studys of Meditation)
#'
#' A shortened dataset from Sedlmeier et al. (2018). The effect of six
#' Meditation studies.
#'
#' @format A data frame with 6 rows and 4 variables:
#' \describe{
#'   \item{Studie}{Author of the study}
#'   \item{Effekt_r}{Effectsize r}
#'   \item{N}{Number of participants}
#'   \item{AV}{Measurrement of the dependent variable}
#'   }
"Meditation"

#' dataset without publication bias
#'
#' A fictitious dataset with effectsizes from 56 studies.
#'
#' @format A data frame with 56 rows and 3 variables:
#' \describe{
#'   \item{N}{Number of participants}
#'   \item{r}{Effectsize r}
#'   \item{p}{p-value (two.tailed)}
#'   }
"pub_bias0"

#' A dataset with publication bias
#'
#' A dataset generated fram pub_bias0 with no nonsignificant p-values.
#'
#' @format A data frame with 28 rows and 3 variables:
#' \describe{
#'   \item{N}{Number of participants}
#'   \item{r}{Effectsize r}
#'   \item{p}{p-value (two.tailed)}
#'   }
"pub_bias1"

#' pedersen_2002
#'
#' Number of desired sexual partners
#'
#' @format  A dataset of 206 persons and their desired number of sexual partners.
#' A modified Version of Pedersen et al. (2002).
#' \describe{
#'   \item{person_id}{ID of participants}
#'   \item{desired}{Number of desired sexual partners}
#'   \item{sex}{male vs. female}
#'   }
"pedersen_2002"

#' VL_17
#'
#' Preference for cats and dogs
#'
#' @format  A dataset of 50 preference ratings for cats and dogs.
#' \describe{
#'   \item{Hunde}{Preference for dogs}
#'   \item{desired}{Preference for cats}
#'   }
"VL_17"

#' Mehrere AVs in einem Einzelfall
#'
#' @format  Erwins Daten aus einem AB-Einzelfalldesign
#' \describe{
#'   \item{Fall}{Name der ProbandInnen}
#'   \item{Phase}{Phase im Einzelfalldesign}
#'   \item{Pos_Emot}{Positive Emotionen}
#'   \item{Neg_Emot}{Negative Emotionen}
#'   \item{Angst}{Angstrating}
#'   \item{Depression}{Depressionrating}
#'   }
"Mehrere_AVs"

#' 6 Meditierende im Multiple Baseline Design (scan-Paket)
#'
#' @format  6 TeilnehmerInnen aus dem Multiple Baseline Experiment von Matko
#' et al. 2021 zur Wirkung von Meditation als scan-Objekt
#' \describe{
#'   \item{baseline}{Dauer der Baseline (d)}
#'   \item{study_day}{Studientag}
#'   \item{wellbeing}{Wohlbefinden}
#'   \item{phase}{Phase im Multiple Baseline Design}
#'   }
"Wellbeing_Daten"

#' Lebenszufriedenheit von 6 Einzelfällen
#'
#' 6 Meditierende im Multiple Baseline Design (data.frame)
#'
#' @format  6 TeilnehmerInnen eines multiple Baseline Experiment von Matko
#' et al. 2021 zur Wirkung von Meditation als data.frame im long-Format.
#' \describe{
#'   \item{baseline}{Dauer der Baseline (d)}
#'   \item{study_day}{Studientag}
#'   \item{wellbeing}{Wohlbefinden}
#'   \item{phase}{Phase im Multiple Baseline Design}
#'   }
"Wellbeing_kurz"

#' Befragung zur Arbeitszufriedenheit
#'
#' 176 Arbeitskräfte beantworten 10 Items zur Arbeitssituation.
#'
#' @format  176 Fälle bei 10 Variablen (data.frame)
#' \describe{
#'   \item{S1}{Mein Arbeitgeber achtet auf einen ordnungsgemäßen Arbeitsplatz.}
#'   \item{S2}{Mein Arbeitgeber bietet mir die modernste / aktuellste Sicherheitsausstattung.}
#'   \item{S3}{Mein Arbeitgeber hat Arbeitssicherheit Vorrang vor der Produktion.}
#'   \item{Q1}{In meinem Betrieb geht Produktqualität vor Quantität.}
#'   \item{Q2}{In meinem Betrieb wird Ressourcen Verschwendung vermieden.}
#'   \item{Q3}{In meinem Betrieb versuchen erkrankte Mitarbeiter so schnell wie möglich an den Arbeitsplatz zurückzukehren.}
#'   \item{Z1}{Ich bin zufrieden mit dem Organisationsklima und dem Teamgeist.}
#'   \item{Z2}{Ich bin zufrieden mit den Arbeitsbedingungen.}
#'   \item{Z3}{Ich bin zufrieden mit den Karrieremöglichkeiten in meinem Betrieb.}
#'   \item{Lohn}{Wohlbefinden}
#'   }
"Arbeit"
